packages/terra-date-picker/docs/DatePickerEvents.md

```diff
@@ -0,0 +1,27 @@
+# Terra DatePicker - Events
+
+## Events
```

Comment:
nit: to be concise, either remove this header or remove "Events" on line 1

Reply:
Updated: https://github.com/cerner/terra-framework/pull/588/commits/608599424d554dff2a4908132605c72032d51661